README.md

```diff
@@ -25,9 +25,11 @@ Usage of multiple file types in the same module will require
 qualified imports. Please see `src/Pangraph/Examples` for further examples.
 
 ## Building
+Using cabal, dependencies track stack lts where possible.
 ```haskell
-stack build
-stack test
+cabal update
+cabal new-build
+cabal new-test
 ```
```

Comment:
It may be useful to keep instructions for Stack users too, especially since I think `algebraic-graphs-0.2` is currently not in LTS (we'll need to fix this).

Reply:
It seems running `stack init` can generate a build plan so I have added this.